Branch-SDK/Branch-SDK/Requests/BranchOpenRequest.m

```diff
@@ -214,25 +215,25 @@ + (void) initialize {
 + (void) setWaitNeededForOpenResponseLock {
     @synchronized (self) {
         if (!openRequestWaitQueueIsSuspended) {
-            //NSLog(@"Suspend openRequestWaitQueue.");
+            BNCLogDebug(@"Suspend openRequestWaitQueue.");
```

Comment:
I'd argue that all of the logs in this class are meaningless to the developer and should not be here. They should have been deleted prior to feature release.